### ISSUE TITLE
Opening quickfix list permanently increases height of bottom layout

### ISSUE THREAD

**Issue by albertfgu** [rcarriga/nvim-dap-ui#268]
I'm using the default layout which puts the REPL and console at the bottom. When I open the quickfix list (or other similar types of small windows which open at the bottom), they bump the REPL/console up which keeps the same height. However then when I close the quickfix window, the top of those windows remains at the same location.

Quick visualization

Start:
```
------------
file


------------
repl | console
------------
```
After `:copen`

```
------------
file
------------
repl | console
------------
quickfix
------------
```
After closing

```
------------
file
------------
repl | console


------------
```

**Comment by rcarriga** [comment]
You can call `dapui.open({ reset = true })` to reset the windows to the default size

**Comment by pferraggi** [comment]
indeed, it would be nice if the ui would just keep the size after somthing else is open or closed, nvim-tree, toggleterm, neotest, etc. currently I have to keep opening and closing the ui to get it back to its correct size

**Comment by Raagh** [comment]
sorry to bring back this but it is annoying that is still happening, it is also weird that this is the only plugin that does this. opening neotest and neo-tree, no increase, opening neotest, toggleterm no increase, opening dap-ui and then whatever else you want, expands the ui.

**Comment by serranomorante** [comment]
Hey, maybe https://github.com/kwkarlwang/bufresize.nvim could help with this.

**Comment by lukealvoeiro** [comment]
Also noticing this issue.
